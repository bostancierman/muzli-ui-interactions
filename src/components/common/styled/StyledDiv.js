import styled from 'styled-components'
import { map } from 'react-immutable-proptypes'
import { Map } from 'immutable'

export const StyledDiv = styled.div`
  ${props => props.css.toJS()}
`

StyledDiv.defaultProps = {
  css: Map()
}

StyledDiv.propTypes = {
  css: map
}

export default StyledDiv
