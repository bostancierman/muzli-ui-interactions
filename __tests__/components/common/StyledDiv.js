import React from 'react'
import StyledDiv from 'src/components/common/styled/StyledDiv.js'

it('renders correctly', () => {
  expect(<StyledDiv>Boilerplate</StyledDiv>).toMatchSnapshot()
})
