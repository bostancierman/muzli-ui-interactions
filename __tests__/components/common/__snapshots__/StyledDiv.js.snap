// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.div
  css={Immutable.Map {}}
>
  Boilerplate
</styled.div>
`;
